package.json

```diff
@@ -34,8 +34,8 @@
     "extends @nextcloud/browserslist-config"
   ],
   "engines": {
-    "node": "^16.0.0",
-    "npm": "^7.0.0 || ^8.0.0"
+    "node": "^20.0.0",
```

Comment:
@skjnldsv I just saw 20 is not LTS yet? So why upgrade already?

Reply:
Because this is for nc 28, and LTS will be Node 20 when we relelase

Reply:
Ah I see